htsget_server/drs_operations.py

```diff
@@ -34,15 +34,15 @@ def get_service_info():
 
 
 @app.route('/ga4gh/drs/v1/objects/<path:object_id>')
-def get_object(object_id, expand=False):
+def get_object(object_id, expand=False, allow_elevated_privileges=False):
```

Comment:
The way I usually do this is that I invoke database.get_drs_object instead of the endpoint if I want to do it without going through authz.